cupsfilters/image-jpeg-xl.c

```diff
@@ -39,7 +39,7 @@ _cfIsJPEGXL(const unsigned char *header, size_t len)
  * Returns: Pointer to a cf_image_t on success, or NULL on failure.
  */
 static cf_image_t *
-cf_image_create_from_jxl_decoder(JxlDecoder *decoder)
+cfImageCreateFromJxlDecoder(JxlDecoder *decoder)
 {
```

Comment:
Please undo this renaming. This is a **static** function, valid only within the file where it is implemented. So its name should be underscore-separated lowercase, and your original name was already correct. Keep it named `cf_image_create_from_jxl_decoder()` .